nav2_collision_monitor/src/polygon.cpp

```diff
@@ -258,6 +258,8 @@ bool Polygon::getCommonParameters(std::string & polygon_pub_topic)
       action_type_ = SLOWDOWN;
     } else if (at_str == "approach") {
       action_type_ = APPROACH;
+    } else if (at_str == "do_nothing") {
```

Comment:
`none` would be cleaner looking

Reply:
to be clear, we're talking only about the param string right? We agree to keep the type as DO_NOTHING and not create an extra type?
About the actual, I have no strong opinion, let me know.

Reply:
Yeah, just the parameter name. `notify` would be fine too

Reply:
> We agree to keep the type as DO_NOTHING and not create an extra type?

I think, that should be enough in current design: if we are not going to support approach notification type, it could just re-use `DO_NOTHING` action type for notification in `CollisionDetector` (otherwise, this better to take `STOP` and `APPROACH` values).

Reply:
Just needs the docs / readme update on adding the new param option